plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/ExBudgetingDefaults.hs

```diff
@@ -236,3 +322,9 @@ unitCostBuiltinCostModel = BuiltinCostModelBase
     , paramByteStringToInteger             = unitCostTwoArguments
     }
 
+unitCekParameters :: Typeable ann => MachineParameters CekMachineCosts DefaultFun (CekValue DefaultUni DefaultFun ann)
```

Comment:
Since this is only by the uplc executable, I would suggest to move it there (if it is possible)